fontforge/splineutil.c

```diff
@@ -3014,7 +3014,7 @@ SplineFont *SplineFontFromPSFont(FontDict *fd) {
 	SplineFontMetaData(sf,fd);
 	if ( fd->wascff ) {
 	    SplineFontFree(sf);
-	    sf = fd->sf;
+return ( NULL );
```

Comment:
We're not doing the unindented returns anymore. Some of those are left over from the old days. 

Still, this may not be quite right. I assume the problem is that `fd->sf` can be NULL, and then (for example) the sf->multilayer check below will fail? That's true, but in that case shouldn't we first check if fd->sf is NULL, return if it is, and otherwise do what's there now?